app/src/main/java/com/github/se/travelpouch/model/gps/GPSRepository.kt

```diff
@@ -24,7 +24,7 @@ class GPSRepository(private val fusedLocationClient: FusedLocationProviderClient
 
   companion object {
     private const val LOCATION_UPDATE_INTERVAL = 10000L // 10 seconds
```

Comment:
I've already thought about it. I don't know what the real impact of using GPS is on the battery. When I did the research it was recommended to use 5 s. I therefore increased it to 10 s for this worry on the battery. I don't think it would be wise to increase it more because for real-time location 30 seconds is huge, especially if you're in a car